### PR TITLE
replace validator percentage with SUI

### PR DESCRIPTION
**Pull request by Jibz1**
Screenshot 2022-12-12 at 5 38 37 PM

### CODE REVIEW

File: apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ function StakeColumn(prop: { stake: bigint; stakePercent: number }) {
                 {prop.stake.toString()}
             </Text>
             <Text variant="captionSmall" color="steel-dark">
-                {prop.stakePercent.toFixed(2)}%
+                SUI
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:126]: I don't think this is really true, we need to use `useFormatCoin` to correctly denominate the SUI.

Comment (Jibz1) [reply apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:126]: Got it

Comment (Jibz1) [reply apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:126]: is it save to use `0x2::sui::SUI` as the coin type

Comment (Jordan-Mysten) [reply apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx]: Yeah I think it is

File: apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx [hunk 2]
```diff
@@ -15,6 +16,7 @@ import { PlaceholderTable } from '~/ui/PlaceholderTable';
 import { TableCard } from '~/ui/TableCard';
 import { Text } from '~/ui/Text';
 
+const COIN_TYPE = '0x2::sui::SUI';
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:19]: Can also just use `SUI_TYPE_ARG` from the SDK.

File: apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx [hunk 3]
```diff
@@ -116,14 +118,16 @@ export type ValidatorState = {
     };
 };
 
-function StakeColumn(prop: { stake: bigint; stakePercent: number }) {
+
+function StakeColumn({ stake, stakePercent }: { stake: bigint; stakePercent: number }) {
```
Comment (Jordan-Mysten) [inline apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:122]: Remove stakePercent?

Comment (Jibz1) [reply apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:122]: I think we may revert back to this later

Comment (Jordan-Mysten) [reply apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:122]: We can always add it back when we want, I'd rather the code reflect the current state of the app not potential future states

Comment (Jordan-Mysten) [reply apps/explorer/src/components/top-validators-card/TopValidatorsCard.tsx:122]: We can keep the calculation in place, just remove this prop

**Review by Jordan-Mysten — APPROVED** [review]

**Pull request merged**